src/payment/strategies/googlepay/googlepay-adyenv2-processor.ts

```diff
@@ -0,0 +1,102 @@
+import { getScriptLoader, getStylesheetLoader } from '@bigcommerce/script-loader';
+import { some } from 'lodash';
+
+import { Payment, PaymentActionCreator, PaymentInitializeOptions } from '../..';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType, RequestError } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+import { AdyenAction, AdyenAdditionalAction, AdyenAdditionalActionState, AdyenClient, AdyenError, AdyenV2ScriptLoader } from '../adyenv2';
+
+export default class GooglePayAdyenV2Processor {
```

Comment:
To keep things consistent please rename this class to `GooglePayAdyenV2PaymentProcessor` and file name to `googlepay-adyenv2-payment-processor`